src/App.js

```diff
@@ -31,6 +31,7 @@ function App() {
           <option value="nl-NL">Nederlands</option>
           <option value="fr-FR">French - FR (Français)</option>
           <option value="pt-BR">pt-BR</option>
+          <option value="tr-TR">Türkçe</option>
```

Comment:
To help out the effort in https://github.com/bndw/wifi-card/pull/81, this should read `Turkish (Türkçe)`